structured_experiments/README.md

```diff
@@ -0,0 +1,33 @@
+#Structured Experiments
+##How to use the template folder
+- Copy the template folder within your branch
+- Rename the folder to YYYY.MM.DD.<my_experiment> where <my_experiment> is the name of your experiment
+- Add a brief (1 sentence) description at the top of the experiment.md file replacing the sample after DESC
+- Create and/or include any files needed within the folder except data files
+  - Data is set up in the docker compose file with different data streams being managed using git lfs
+  - When using a different data set the volume must be set to that directory in the docker-compose under the volume label
+- Edit the Dockerfile, docker-compose, experiment.sh, requirements files as needed for repeatability
+- Run `docker-compose -f docker-compose.yml build train && docker-compose -f structured_experiments/2022.02.01.test/docker-compose.yml up --build train_test_experiment` to train a tesseract model

```

Comment:
The local folder could be commented to reflect the local experiment with generic info for clarity, (.../YYYY.MM.DD/test_name/docker-compose.yml...)

Reply:
Good catch. I'll get that updated.